### DISCUSSION TITLE
Pallas Installation

### DISCUSSION THREAD

**Discussion by karan-dalal** [jax-ml/jax#19154] [Q&A] [answered]
I'm trying to get started with Pallas, but I'm unable to get a correct installation. Following a [previous issue](https://github.com/google/jax/discussions/17367) doesn't work as it appears the packages are outdated.

Is there an updated installation guide for Pallas?

**Comment by jakevdp** [comment, marked as answer]
Hi - thanks for the question! This is still relatively tricky, unfortunately: we're actively working on improving the Pallas installation experience; work is tracked in https://github.com/google/jax/issues/18603.
